t/cli/test_etcd_sync_event_handle.sh

```diff
@@ -127,7 +127,7 @@ cat logs/error.log | grep "watchdir err: has no healthy etcd endpoint available"
 ##   }
 ## }
 ## After check, it only appears when watch recovers and returns events in bulk.
-cat logs/error.log | grep "}, {" || (echo "failed: Log case 2 unexpected"; exit 1)
+cat logs/error.log | grep "etcd events sent in bulk" || (echo "failed: Log case 2 unexpected"; exit 1)
```

Comment:
Since the entire res object is no longer logged. We programmatically assert in code and log this specific condition to make this test case pass.